extendr-api/src/wrapper/matrix.rs

```diff
@@ -430,12 +478,13 @@ impl<T, D> DerefMut for RArray<T, D> {
 
 #[cfg(test)]
 mod tests {
+    use super::*;
+    use crate as extendr_api;
     use extendr_engine::with_r;
     use prelude::{Rcplx, Rfloat, Rint};
 
-    use super::*;
-
     #[test]
+    #[ignore]
```

Comment:
why do we ignore this test?

Reply:
I don't need to ignore this. Removed it. And pushed.